API/src/main.ts

```diff
@@ -1,8 +1,44 @@
 import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import * as bodyParser from 'body-parser';
+import * as helmet from 'helmet';
 
-async function bootstrap() {
+import { INestApplication } from '@nestjs/common';
+import { AppModule } from './app/app.module';
+
+const API_DEFAULT_PORT = 4000;
```

Comment:
unless there's some very clear documentation, we should not have a default hardcoded - and i don't see any documentation changes in this PR